VideoWeb/VideoWeb.EventHub/Handlers/Core/EventHandlerBase.cs

```diff
@@ -115,6 +114,19 @@ await HubContext.Clients.Group(Hub.EventHub.VhOfficersGroupName)
                             $"Endpoint State: { endpointState } | Timestamp: { (DateTime.Now) :yyyy-MM-dd HH:mm:ss.fffffff} ");
         }
 
+        protected async Task PublishRoomTransferMessage(RoomTransfer roomTransfer)
+        {
+            foreach (var participant in SourceConference.Participants)
+            {
+                await HubContext.Clients.Group(participant.Username.ToLowerInvariant())
+                    .RoomTransfer(roomTransfer);
+                Logger.LogTrace($"Room Transfer: Conference Id: { SourceConference.Id } | Participant Id: { participant.Id } | " +
```

Comment:
Could we add transfer from and to also to the logs?

Reply:
ill update and add logging to the class
Ive switched out the message to something more descriptive
`Logger.LogTrace("RoomTransfer sent to group: {group} | Role: { participantRole }", participant.Username, participant.Role);`
The additional properties (and more) are added to the scope automatically by the logging middleware so we dont have to add to the message